### PR TITLE
I75 create fixtures page

### LINKED ISSUE #75: Create Fixtures Page
## Basic Information
Structure: 
tournament -> levels(junior, novice, senior) -> Divisions -> Matchups
where a level have many rounds, divisions have many matchups
- Matchups to include team info (for affirmative and negative) , venue, venue room, time, date, adjudicator(s), round number
- Every collection should all have an id

### Tips
1. create dummy json data with the structure above
2. Make the home page tournaments use the dummy data from the json file
3. Use dynamic pages/routing when a tournament is clicked
4. Follow figma design - Nothing should be hard coded, it must be rendered based on the json data

glhf :) Please speak with committee, or project helpers if you are stuck.

WIP
```
tournaments: [{         
        id,      
        name,  
        short name, 
        (any other fields that might be needed)         
        levels: {
              Junior :{
                Divisions: [{
                    Division number,  
                    Teams,
                    Number of Rounds,
                    Matchups : [{
                         round number,
                         team affirmative,
                         team negative,    
                         venue, datetime, 
                         ajudicators: [{...}]
                         status(?)(upcoming,complete(?))
                    }, ....]
                }, ... ]         }
               Senior: {...}
               Noive: {...}
     }
},
{tourny #2},....]
```

### PR DESCRIPTION
**Pull request by DataScientistDave**
## Change Summary

- Used dynamic routes to create the fixtures page for each tournament. 
- Added the dynamic routes for each fixtures page to the public routes in the middleware.
- The fixtures page generates a table after a round and level is selected.
- The columns of the table are Div, Venue, Date, Time, Affirmative Team, Negative Team, Topic and Status.

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information

# Related issue

- Resolve #75

### CODE REVIEW

File: pages/fixtures/[tournamentId].vue [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    round++;
+  }
+};
+
+const handleLevel = (tabName) => {
+  levelSelected = tabName;
+  levelTabsKey++;
+  tableData.value = [];
+  roundTabs.forEach((round) => (round.active = false));
+  roundSelected = undefined;
```
Comment (ChantelleKerr) [inline pages/fixtures/[tournamentId].vue:78]: It should load the current round as well when changing levels :)

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Hey Nisarg, everything looks really good! Ij ust need one line changed :D
Thanks for all your work on this, amazing stuff.

File: pages/fixtures/[tournamentId].vue (hunk 1 above)
Comment (DataScientistDave) [reply pages/fixtures/[tournamentId].vue]: Should be working now.

**Review by ChantelleKerr — APPROVED** [review]
Great job! It looks really good :D

**Pull request merged**